modules/standard/FileSystem.chpl

```diff
@@ -27,7 +27,7 @@ use Error;
    Note: this is not safe within a parallel context.  A chdir call in one task
    will affect the current working directory of all tasks for that locale.
 */
-proc locale.chdir(out err: syserr, name: string) {
+proc locale.chdir(out error: syserr, name: string) {
```

Comment:
Sorry, I'm not being clear.  The uses of the argument within the function itself have not been updated.